app/src/pages/inside/common/utils.js

```diff
@@ -157,8 +157,9 @@ export const getGroupedDefectTypesOptions = (
     });
     defectTypesOptions = defectTypesOptions.concat(
       defectTypeGroup.map((defectType) => {
-        const label = defectTypesLocalization[defectType.typeRef.toLowerCase()]
-          ? formatMessage(defectTypesLocalization[defectType.typeRef.toLowerCase()])
+        const defectTypeLabelId = `Defect_Type_${defectType.locator.toUpperCase()}`;
+        const label = defectTypesLocalization[defectTypeLabelId]
+          ? formatMessage(defectTypesLocalization[defectTypeLabelId])
```

Comment:
Why is this all, if you can just 
label: defectType.longName,

Reply:
It's done for default defect types translation